### PR TITLE
added navabr

### LINKED ISSUE #53: Header / Navbar
**Deliverables**
- Add Navbar Component 
- Each Button/Link has a hover effect.
- Logo `Assets/GS_logo_black (1).png`

**Sample**
Capture

**Additional context**
- Tech/Framework:- React Js

### PR DESCRIPTION
**Pull request by shreyanshi16**
# Description

Added Navbar

Closes #53 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

https://user-images.githubusercontent.com/53462278/113285935-2b936200-9309-11eb-88d4-bac97ea0b8b7.mp4

### CODE REVIEW

File: frontend/src/Components/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        </Button>
+        <Button className="btn" type="submit">
+          Join Us
+        </Button>
+      </div>
+    </nav>
+  );
+};
+
+const Button = styled.button`
```
Comment (codewithvk) [inline frontend/src/Components/Navbar/Navbar.js:63]: Please write CSS code in `navbar.css`

**Review by codewithvk — CHANGES_REQUESTED** [review]
@shreyanshi16 Make these changes!

File: frontend/src/Components/Navbar/Navbar.js (hunk 1 above)
Comment (shreyanshi16) [reply frontend/src/Components/Navbar/Navbar.js]: Done!

**Comment by codewithvk** [comment]
@shreyanshi16 Please resolve conflict.

**Comment by shreyanshi16** [comment]
@codewithvk conflict resolved

**Comment by codewithvk** [comment]
Thanks for PR @shreyanshi16

**Pull request merged**